chapters/memory-layout/introduction-to-GDB/reading/README.md

```diff
@@ -0,0 +1,84 @@
+# GNU Debugger (GDB)
+
+## Starting GDB
+
+GDB is a powerful tool for debugging programs.
+It allows you to inspect the state of a program at a certain point in its execution, set breakpoints, and step through the code, among other things.
+To start GDB, you need to run the following command:
+
+```bash
+gdb [program_name]
+```
+
+## Running the Program
+
+To run the program being debugged, there are two available commands:
+
+- `r` or `run` - this command will run the program
+- `start` - unlike `run`, this command will start the program but immediately stop after entering `main`
+
+## Breakpoints
+
+The essential element of GDB is the breakpoint.
+Essentially, setting a breakpoint at a certain instruction causes the program's execution to halt every time it reaches that point.
+Setting a breakpoint is done with the following command:
+
+```gdb
+break [location]
```

Comment:
I think that the option of using b [location] should also be included as it is for every other command.